src/hotspot/share/runtime/arguments.cpp

```diff
@@ -1655,11 +1655,12 @@ jint Arguments::set_ergonomics_flags() {
   return JNI_OK;
 }
 
-julong Arguments::limit_by_allocatable_memory(julong limit) {
+julong Arguments::limit_heap_by_allocatable_memory(julong limit) {
```

Comment:
If we are already messing with the name in this change, would it be possible to change return value and parameter to `size_t`? I'm good with moving this change of the parameter types to a different issue, but as mentioned, since we are already changing stuff around here...

Reply:
I'd prefer to handle that as a separate issue, since such a change would propagate into `os::has_allocatable_memory_limit()` and affect all os:es.

Reply:
okay.